tensorflow/python/autograph/operators/BUILD

```diff
@@ -114,6 +114,7 @@ py_test(
         "//tensorflow/python/autograph/core",
         # TODO(b/145618471): Remove this transitive dependency.
         "//tensorflow/python/distribute:input_lib",
+        "//tensorflow/python/ops/parallel_for:control_flow_ops",
```

Comment:
Can you try to also include `//third_party/tensorflow/python/ops/signal`?

Reply:
Also, please be sure to indent consistently, otherwise the lint checks will fail.

Reply:
Thanks @mdanatg , the PR has been updated.